Rokas/Board.java

```diff
@@ -0,0 +1,65 @@
+
+import java.util.Random;
+
+public class Board {
+    Constants constants;
+    Logic logic;
+    Renderer renderer;
+
+    private final Tile[][] matrix;
+    Random rand = new Random();
+    private static Board instance;
+
+
+    private Board(Constants constants, Logic logic, Renderer renderer) {
+        this.constants = constants;
+        this.logic = logic;
+        this.renderer = renderer;
+        matrix = new Tile[4][4];
+        createEmptyBoard();
+        addTile();
+        addTile();
+    }
+
+    public void createEmptyBoard(){
+        for(int i=0; i< matrix.length; i++){
+            for(int j=0; j< matrix.length; j++){
+                matrix[i][j] = new Tile(false);
+            }
+        }
+    }
```

Comment:
Šį metodą galima išskaidyti į kelis atskirus, būtų geresnis skaitomumas.
Pvz:
`public void createEmptyBoard(){
for(int i=0; i< matrix.length; i++){
  createBoardCollumns();
}
}`